STABLE.md

```diff
@@ -0,0 +1,5 @@
+# Supported stable branches
+
+The following stable branches are kindly maintained by trusted volunteers:
+
+* [4.14](https://www.alejandro-colomar.es/src/alx/shadow/stable/shadow.git/log/?h=4.14.x)
```

Comment:
If there is an official stable branch, please keep it in the main repository which is currently on GitHub.
I assume the official 1.14.1, etc. releases will still be released on GitHub.
What would be the point of having an external repository for a stable branch?

Reply:
> My reticence to use GitHub. :)

IMO it hurts the project.

Reply:
What is cheese?

Reply:
> Please detail your opinion. I don't agree, since I'm only adding cheese to the project. I provide support to branches that previously had no support.

I'm sorry, but I'm not going to be engaged in another endless thread, like https://github.com/shadow-maint/shadow/issues/684 .

I believe official stable branches should be maintained with even more care and with at least the same level of testing (infrastructure) as the master branch and having a single maintainer and the very basic external git hosting infrastructure provide none of those.